sigs/contributor-strategy.md

```diff
@@ -0,0 +1,173 @@
+include:  
+1. Maintainers Circle
+2. Contributor growth and outreach: docs, diversity, recruitment, retention
+ * includes modern mentoring, succession planning, and staffing contributor role strategies
+3. "Community/Contributor health check"
+  * evaluation criteria  
+  * check-in/review/consulting process  
+4. Open governance guidelines and governance operations best practices
+  * the why, how, and where, your contributors make decisions  
+  * contributor diversity
+
+*Possible future roadmap projects*  
+If you see something here that interests you, join us and start it:  
+* Training: leadership, code of conduct, code reviewing, etc  
+* Contributor metrics and definitions  
+* Automation and self service for contributors, community GitOps
+
+## Governance
+This SIGs topic requires cross collaboration between end users, CNCF SIGs, and
+CNCF projects of all graduation levels.
+
+This SIG should be populated and governed by reps from CNCF projects that want
+to create and run intentional contributor experience programs, a rep(s) from the
+end user committee, and the TOC liaison(s). While the SIG reps do not need to be
+core maintainers, they do need to have a drive for making things better for
+contributors and end users. We welcome industry experts and academics in relevant working groups!
+
+Conduct public monthly meetings as a large group and working groups that form will
+have their own meetings. (see #Meetings and Decisions for more)
+
+Check in with the TOC on a monthly basis for a quick overview and challenges
+during a public TOC meeting.
+
+### Members
+
+Members are active participants in the work of the SIG who are entitled to vote
+in any SIG decisions that require a vote.  Any contributor to the SIG is
+eligible to become a member after participating in the work of the SIG for at
+least three months.
+
+In order to prevent the SIG from becoming unbalanced, it will have the following
+limits on who can be a voting member:
+
+Up to one from each participating Incubating or Sandbox CNCF project
+Up to two from each Graduated CNCF project
+One from each SIG-ContribStrat Working Group, generally the lead for that WG
+No more than ⅓ of members from the same employer
+
+If a contributor would be entitled to be a member, but are restricted because of
+the above limits, they are a non-voting member who may participate in meetings
+by cannot vote.
+
+Members who are no longer participating actively in the SIG (including both WG
+  work and the regular meetings) will step down from membership.
+
+#### Chairs and TOC Liaison
+
+- TOC Liaison:  
+- Chairs:  
+
+In accordance with the terms and roles laid out in [cncf-sigs.md](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md)
+
+At least one member or contributor is elected to report the SIG's decisions and
+activities to the TOC by an annual preference vote of the current members.  This
+person is known as the "TOC Liaison".
```

Comment:
Terminology could be confusing here. For other SIGs a [TOC Liaison](https://github.com/cncf/toc/tree/master/sigs#current-sigs) is a member of the TOC who works with that SIG (btw we don't have a written down definition of that role yet, but we should)